integration-cli/docker_cli_inspect_test.go

```diff
@@ -9,36 +9,38 @@ import (
 	"time"
 
 	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/pkg/integration/checker"
```

Comment:
I would say yes, (even though the package might change, we'll alias it). Not sure what you mean by "deprecate `go-check`", it still used for everything, just that we use other checkers (complementary checkers) :wink: .